### ISSUE TITLE
Is it possible to move/resize panels and notest?

### ISSUE THREAD

**Issue by ProkopHapala** [spheras/desktopfolder#304]
This seems like exactly what I need (better alternative to screenlets folderview)

but I cannot move and resize folders and panels by my mouse. Is this functionality still missing or it does not work on my system?

( Ubuntu 18.04 with Xfce 4.12 )

Also, I don't know how to  link folder panel to some existing folder on my computer (it should show content of existing panel e.g. of "documents") ... it is perhaps not engouh to copy `.dosktopfolder` file there ?

**Comment by spheras** [comment]
Hi @ProkopHapala, you should be able to move and resize folders and panels. The problem is that the Linux is ecosystem is wide, and it is impossible to check it everywhere.  Probably the problem is XFce integration.  I could help you to detect/solve the problem, but you would need to go deeper into the code. Maybe you could try Ubuntu budgie, as it seems to work properly there.

Regarding the other question. Yes, you can also create links to existent folders. Just right click over the desktop -> new -> Link to folder (or similiiar, I don't remember the exactly words in English).

Hope it helps. Regards.

**Comment by ProkopHapala** [comment]
Thank you, "Link Panel" is exactly what I need!

I'm happy to try debug it on Ubuntu+Xfce, just I have no experience with Vala language, I don't have prepared development environment for it, and not sure where to start.

I think Xfce specifically is good use-case for desktopfolder because there is some long-standing problem with desktop icon on Xfce (https://forum.xfce.org/viewtopic.php?id=7597) which render them almost un-usable, and they were not able to repari it since 2016. That is why I tried screenlets.

Otherwise, I find Xfce the best desktop (I migrated recently from Gnome Classic/Flashback, and tried KDE Plasma, Bugie and some other on the way). All desktops nowadays dumb-down (remove features, to be simple), making large buttons (perhaps touch-friendly), using screen space inefficiently - they are made for users with simple work-flow which does not have to manage large number of files and applications.

**Comment by spheras** [comment]
Yes, that's the problem and the beauty of linux, a lot of choices for users is bad for developers, :smile: 
Anyway, if you feel you can try to debug, I could try to help you, everything is more than welcome... the problem is always "time", at least in my case, it is difficult to find it to dedicate to personal open source projects.
